torch/_inductor/decomposition.py

```diff
@@ -369,6 +369,16 @@ def _foreach_addcdiv_scalar(self, left_tensors, right_tensors, scalar=1):
     )
 
 
+@register_decomposition(aten._foreach_lerp.Scalar)
+def _foreach_lerp_scalar(start_tensors, end_tensors, weight):
+    return aten._foreach_add.List(
+        start_tensors,
+        aten._foreach_mul.Scalar(
+            aten._foreach_sub.List(end_tensors, start_tensors), weight
+        ),
+    )
```

Comment:
Since we are fusing these things into a kernel, perhaps we could use tricks similar to those we use in `linspace` to improve the accuracy. See for example
https://github.com/pytorch/pytorch/blob/ccdc642255d8bc2640529439ba97859d6048f218/torch/_refs/__init__.py#L4618-L4623

cc @janeyx99